Pony/include/pony/Address.hpp

```diff
@@ -9,34 +9,31 @@
 #ifndef PONY_ADDRESS_HPP
 #define PONY_ADDRESS_HPP
 
-#include <pony/SharedMacros.hpp>
+#include <cstdint>
 
 namespace pony
 {
 
-class PONY_EXPORT Address
+class Address
```

Comment:
This looks like it's ipv4 only. What about ipv6?

Reply:
? for now - yep..